package.json

```diff
@@ -2,9 +2,17 @@
   "name": "reactjs2020q2",
   "version": "1.0.0",
   "description": "",
-  "main": "src/server/app.js",
+  "main": "src/server/index.js",
   "scripts": {
-    "start": "node src/server/app.js"
+    "build:dev": "webpack",
+    "build:prod": "NODE_ENV=production webpack",
+    "eslint": "eslint ./src/client --ext .js,.jsx",
+    "start": "npm run build && node src/server/index.js",
+    "client": "concurrently \"webpack-dev-server --hot\" \"watch 'npm run eslint'\"",
+    "server:dev": "nodemon src/server/index.js",
+    "dev": "concurrently \"npm run server\" \"npm run client\"",
+    "prod": "NODE_ENV=production concurrently \"npm run server\" \"npm run client\"",
```

Comment:
I would like to suggest combine and rename some of the commands, it's a little bit confusing naming.

just example:
![image](https://user-images.githubusercontent.com/14198624/81036983-c7beaa00-8ea9-11ea-9c7b-ed1ffe0209e2.png)

Reply:
Fixed this partially. Removed redundant tasks which were not cleared after previous refactoring. For server left for now only 1 task as I don't have any differences for it right now.